### PR TITLE
Eb 45 change profile data

### PR DESCRIPTION
**Pull request by Hesoyam01**

### CODE REVIEW

File: src/app/features/user/components/edit-dialog/edit-dialog.component.ts
Comment (J-C-V) [inline src/app/features/user/components/edit-dialog/edit-dialog.component.ts:1]: - remove debug console.log
- remove unused code (currently commented out)
- dont sign out user after changing email or password successfully

File: src/app/features/user/pages/profile/profile.component.html
Comment (J-C-V) [inline src/app/features/user/pages/profile/profile.component.html:1]: - why is email an a? just move the current email in the edit email dialog
- maybe flex order column and buttons full width instead?

File: src/app/features/user/pages/profile/profile.component.scss
Comment (J-C-V) [inline src/app/features/user/pages/profile/profile.component.scss:1]: - remove unused code

**Review by J-C-V — CHANGES_REQUESTED** [review]
see comments

**Comment by J-C-V** [comment]
also add some sort of feedback for the user - if change was successful or not

**Comment by Hesoyam01** [comment]
@J-C-V ready to merge

**Review by J-C-V — CHANGES_REQUESTED** [review]
See comment

File: src/app/features/user/components/edit-dialog/edit-dialog.component.html
Comment (J-C-V) [inline src/app/features/user/components/edit-dialog/edit-dialog.component.html:1]: - Switch confirm/cancel button position

**Review by J-C-V — CHANGES_REQUESTED** [review]
See comments

**Review by J-C-V — APPROVED** [review]
LGTM

**Pull request merged**